_data/simple-icons.json

```diff
@@ -3075,6 +3075,11 @@
             "hex": "F4731C",
             "source": "http://v.ok.ru/logo.html"
         },
+        {
+            "title": "Opel",
+            "hex": "F7D900",
+            "source": "https://commons.wikimedia.org/wiki/File:Opel-Logo_2017.svg"
```

Comment:
As usual, I'd suggest using the Wikipedia page as the source URL to avoid potential future "link rot".